Tools/ros2/ardupilot_dds_tests/setup.py

```diff
@@ -0,0 +1,34 @@
+import os
+from glob import glob
+from setuptools import setup
+
+package_name = 'ardupilot_dds_tests'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        (os.path.join("share", package_name, "launch"),
```

Comment:
Want to use pathlib?

Reply:
Yes - could do. Pathlib being the modern approach right? By and large I followed exiting examples from the ROS source and the Humble tutorials. `launch` is circa 2015 - so prob a bit out of date in terms of Python flavour.